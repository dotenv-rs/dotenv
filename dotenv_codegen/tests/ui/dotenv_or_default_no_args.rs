#[macro_use]
extern crate dotenv_codegen;

pub fn main() {
    dotenv_or_default!();
}