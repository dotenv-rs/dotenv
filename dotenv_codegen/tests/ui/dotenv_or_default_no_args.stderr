error: proc-macro derive panicked
 --> $DIR/dotenv_or_default_no_args.rs:5:5
  |
5 |     dotenv_or_default!();
  |     ^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: expected 1 or 2 arguments, found none
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: cannot find macro `proc_macro_call` in this scope
 --> $DIR/dotenv_or_default_no_args.rs:5:5
  |
5 |     dotenv_or_default!();
  |     ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
